addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -2,7 +2,11 @@
 
 import weWidgets from 'wysiwyg.widgets';
 import {_t} from 'web.core';
+import { browser } from "@web/core/browser/browser";
 weWidgets.LinkPopoverWidget.include({
+    events: Object.assign({}, weWidgets.LinkPopoverWidget.prototype.events, {
```

Comment:
New line between imports and business code, here you removed it by typing something inside it